docs/en/install-upgrade/installing-stack-demo-secure.asciidoc

```diff
@@ -0,0 +1,894 @@
++
+["source","shell"]
+----
+chown root:elasticsearch *
+----
+
+.. Set the files in `/etc/elasticsearch/certs` to have read and write permissions by the owner (`root`) and read permission by the `elastic` user:
++
+["source","shell"]
+----
+chmod 640 *
+----
+
+.. Change the `/etc/elasticsearch/certs` and `/etc/elasticsearch/certs/ca` directories to be executable by the owner:
++
+["source","shell"]
+----
+chmod 750 /etc/elasticsearch/certs
+chmod 750 /etc/elasticsearch/certs/ca
+----
+
+. Restart the {es} service:
++
+["source","shell"]
+----
+sudo systemctl start elasticsearch.service
+----
+
+. Run the status command to confirm that {es} is running:
++
+["source","shell"]
+----
+sudo systemctl status elasticsearch.service
+----
++
+In the event of any problems, you can also monitor the {es} logs for any issues by tailing the {es} log file:
++
+["source","shell"]
+----
+sudo tail -f /var/log/elasticsearch/elasticsearch-demo.log
+----
++
+A line in the log file like the following indicates that SSL has been properly configured:
++
+["source","shell"]
+----
+[2023-07-12T13:11:29,154][INFO ][o.e.x.s.Security         ] [es-ssl-test] Security is enabled
+----
+
+[discrete]
+[[install-stack-demo-secure-second-node]]
+== Step 4: Configure security on additional {es} nodes
+
+Now that the security is configured for the first {es} node, some steps need to be repeated on each additional {es} node.
+
+. To avoid filename collisions, on each additional {es} node rename the existing `http.p12` file in the `/etc/elasticsearch/certs/` directory:
++
+["source","shell"]
+----
+mv http.p12 http-old.p12
+----
+
+. Copy the CA and truststore files that you generated on the first {es} node so that they can be reused on all other nodes:
+
+* Copy the `/ca` directory (that contains `ca.crt` and `ca.key`) from `/etc/elasticsearch/certs/` on the first {es} node to the same path on all other {es} nodes.
+
+* Copy the `elastic-stack-ca.p12` file from `/etc/elasticsearch/certs/` to the `/etc/elasticsearch/certs/` directory on all other {es} nodes.
+
+* Copy the `http.p12` file from each node directory in `/etc/elasticsearch/certs/elasticsearch/` (that is, `elasticsearch/mynode-es1`, `elasticsearch/mynode-es2` and `elasticsearch/mynode-es3`) to the `/etc/elasticsearch/certs/` directory on each corresponding cluster node.
+
+. On each {es} node, repeat the steps to generate a new certificate for the transport layer:
+
+.. Stop the {es} service:
++
+["source","shell"]
+----
+sudo systemctl stop elasticsearch.service
+----
+
+.. From the `/etc/elasticsearch/certs` directory, create a new certificate for the {es} node:
++
+["source","shell"]
+----
+sudo usr/share/elasticsearch/bin/elasticsearch-certutil cert --ca-cert /etc/elasticsearch/certs/ca/ca.crt --ca-key /etc/elasticsearch/certs/ca/ca.key
+----
++
+When prompted, choose an output file name or use the default, and specify a password for the certificate.
```

Comment:
We should probably suggest using `/etc/elasticsearch/certs/elastic-certificates.p12` as the output file to avoid having to copy files around.